src/ucp/core/ucp_worker.c

```diff
@@ -1234,13 +1240,28 @@ static void ucp_worker_iface_set_sys_device_distance(ucp_worker_iface_t *wiface)
     }
 }
 
+static const ucp_tl_md_t *
+ucp_worker_iface_get_md(const ucp_worker_iface_t *wiface)
+{
+    const ucp_tl_resource_desc_t *rsc = ucp_worker_iface_get_tl_resource(
```

Comment:
do not init inline so the function call (to ucp_worker_iface_get_tl_resource) would not be split to 2 lines